apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsTextControl.tsx

```diff
@@ -62,7 +69,25 @@ export function JsonFormsTextControl({
   return (
     <Box>
       <FormControl isRequired={required} isInvalid={!!errors}>
-        <FormLabel description={description} mb={0} tooltipText={tooltip}>
+        <FormLabel
+          description={<Markdown>{description}</Markdown>}
+          mb={0}
+          tooltipText={tooltip}
+          sx={{
+            "& a": {
+              color: linkColor,
+              textDecoration: "underline",
+              _hover: {
+                color: linkHoverColor,
+                textDecoration: "none",
+              },
+              _active: {
+                color: linkActiveColor,
+                textDecoration: "none",
+              },
+            },
+          }}
+        >
```

Comment:
can do some simple extraction of this into something reusable? IMO this is quite useful and can be used for other stuff

Reply:
i don't mind, but need to be careful about this, because it alters styles in-line (style has higher precedence than classname). 

i'll put as a `MarkdownLabel` for now w/ just the `a` tag styling if you're ok with that?